actions/actions.py

```diff
@@ -501,22 +504,18 @@ def run(
         for intent in first_intent_names:
             logger.debug(intent)
             logger.debug(entities)
-            buttons.append(
-                {
-                    "title": self.get_button_title(intent, entities),
-                    "payload": "/{}{}".format(intent, entities_json),
-                }
-            )
+            if len(intent.split("/")) > 1:
+                message = self.get_button_title(intent, entities)
+                buttons.append({"title": message, "payload": message})
```

Comment:
add a comment about why we have to use the message instead of regex format for retrieval intents